bash_tools/bin/stowsh

```diff
@@ -119,6 +119,12 @@ function _install {
     local dest_filepath
     dest_filepath="${DEST_DIR}/$(basename "${filename}")"
     _log 'install' "creating link: [${dest_filepath}]"
+
+    if [[ -f "${dest_filepath}" ]] || [[ -d "${dest_filepath}" ]]; then
+      _log 'install' "[${dest_filepath} already exists and is not a symlink; skipping"
+      continue
```

Comment:
skipping and warning would be better UX than failing fast, in this case; a dev would very likely have a ton of stuff already there, and we wouldn't want to block them just because of that.